src/server.py

```diff
@@ -215,9 +210,10 @@ def _build_pipeline(self):
         elements = [
             "filesrc name=src location=\"{}\"".format(self.sound_location),
             "decodebin",
-            "volume name=volume volume={}".format(pipeline_volume),
+            "identity single-segment=true",
```

Comment:
This needs an explanation in the commit message of why it works. The documentation of the `identity` element is not very enlightening.

Reply:
The problem I explained in the big comment was supposed to be fixed by just doing a SEGMENT seek. But for some reason the clock kept restarting at the end of the loop. I am not 100% sure yet, but it seems that the decodebin does not handle the SEGMENT seek properly or at least as I expect. Each buffer has a timestamp. I believe the timestamp of each buffer was restarting to 0 at the end of the loop. The "identity" element is adding the correct timestamps to the buffers pushed by the decodebin.